src/routes/index/_about.svelte

```diff
@@ -35,7 +35,7 @@
 				>
 					<h3 class="text-center lg:text-left my-10 lg:my-8">Our journey with Hacktoberfest</h3>
 					<p class="hidden lg:block text-justify pb-[3rem]">
-						Hacktoberfest is an initiative that very deeply matters to us. We launched the first
+						Hacktoberfest is an initiative that matters, very deeply, to us. We launched the first
```

Comment:
```suggestion
						Hacktoberfest is an initiative that matters, very deeply, to us. We launched the first
```
```suggestion
						Hacktoberfest is an initiative that matters very deeply to us. We launched the first
```

Reply:
I have committed these changes, should I make anymore or would you approve of this?